src/mmw/apps/geoprocessing_api/schemas.py

```diff
@@ -178,3 +178,61 @@
     },
     required=['location'],
 )
+
+nlcd_override_allowed_values = '", "'.join([
+    'nlcd-2019-30m-epsg5070-512-byte',
+    'nlcd-2016-30m-epsg5070-512-byte',
+    'nlcd-2011-30m-epsg5070-512-byte',
+    'nlcd-2006-30m-epsg5070-512-byte',
+    'nlcd-2001-30m-epsg5070-512-byte',
+    'nlcd-2011-30m-epsg5070-512-int8',
+])
+LAYER_OVERRIDES = Schema(
+    title='Layer Overrides',
+    type=TYPE_OBJECT,
+    description='MMW combines different datasets in model runs. These have '
+                'default values, but can be overridden by specifying them '
+                'here. Only specify a value for the layers you want to '
+                'override.',
+    properties={
+        '__LAND__': Schema(
+            type=TYPE_STRING,
+            example='nlcd-2019-30m-epsg5070-512-byte',
+            description='The NLCD layer to use. Valid options are: '
+                        f'"{nlcd_override_allowed_values}". All "-byte" '
+                        'layers are from the NLCD19 product. The "-int8" '
+                        'layer is from the NLCD11 product. The default value '
+                        'is NLCD19 2019 "nlcd-2019-30m-epsg5070-512-byte".',
+        ),
+        '__STREAMS__': Schema(
+            type=TYPE_STRING,
+            example='nhdhr',
+            description='The streams layer to use. Valid options are: '
+                        '"nhdhr" for NHD High Resolution Streams, "nhd" for '
+                        'NHD Medium Resolution Streams, and "drb" for '
+                        'Delaware High Resolution. The area of interest must '
+                        'be completely within the Delaware River Basin for '
+                        '"drb". "nhdhr" and "nhd" can be used within the '
+                        'Continental United States. In some cases, "nhdhr" '
+                        'may timeout. In such cases, "nhd" can be used as a '
+                        'fallback. "nhdhr" is the default.'
+        )
+    },
+)
+
+MODELING_REQUEST = Schema(
+    title='Modeling Request',
+    type=TYPE_OBJECT,
+    properties={
+        'area_of_interest': MULTIPOLYGON,
+        'wkaoi': Schema(
```

Comment:
We use a distinct definition here from [this one](https://github.com/WikiWatershed/model-my-watershed/blob/4586d82120d5115dd31ccedfdec545f3172c2dc5/src/mmw/apps/geoprocessing_api/schemas.py#L51-L59), because that is for a query parameter, and here it is a JSON parameter.

Reply:
Nice clarification!